contracts/Vote.sol

```diff
@@ -266,16 +285,18 @@ contract Vote is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed {
     }
 
     /**
-     * @notice Retuns currently locked celo in voting. (This celo cannot be unlocked.)
+     * @notice Updates the beneficiaries voting history and returns locked stCELO in voting.
+     * (This stCELO cannot be unlocked.)
      * And it will remove voted proposals from account history if appropriate.
      * @param beneficiary The beneficiary.
+     * @return Currently locked stCELO in voting.
      */
     function updateHistoryAndReturnLockedStCeloInVoting(address beneficiary)
```

Comment:
Removed `onlyManager` modifier, because there is no real restriction, since any user can directly call this from manager.